src/devices/esp8266.c

```diff
@@ -329,6 +330,9 @@ bool esp8266_init(struct Serial *s, const size_t max_cmd_len,
         if (!_setup(s, max_cmd_len))
                 return false;
 
+        /* Hard reset our ESP8266*/
```

Comment:
The device reset on MK2 will be AT command generated.  We will try to issue the command first, then if that fails fall back to the hard method.  So there will be some form of AT reset on MK2